src/otx/recipe/classification/otx_dino_v2.yaml

```diff
@@ -0,0 +1,68 @@
+# @package _global_
+defaults:
+  - override /base: classification
+  - override /data: mmpretrain
+  - override /model: torch_classification
+data:
+  train_subset:
+    batch_size: 64
+    transforms:
+      - type: LoadImageFromFile
+        to_float32: true
+      - backend: cv2
+        scale: 224
+        type: RandomResizedCrop
+      - type: PackInputs
+  val_subset:
+    batch_size: 64
+    transforms:
+      - type: LoadImageFromFile
+        to_float32: true
+      - backend: cv2
+        edge: short
+        scale: 256
+        type: ResizeEdge
+      - crop_size: 224
+        type: CenterCrop
+      - type: PackInputs
+  test_subset:
+    batch_size: 64
+    transforms:
+      - type: LoadImageFromFile
+        to_float32: true
+      - backend: cv2
+        edge: short
+        scale: 256
+        type: ResizeEdge
+      - crop_size: 224
+        type: CenterCrop
+      - type: PackInputs
+model:
+  otx_model:
+    _target_: otx.algo.classification.otx_dino_v2.DINOv2RegisterClassifier
+    config:
+      backbone:
+        name: dinov2_vits14_reg
+        frozen: true
+      head:
+        in_channels: 384
+        num_classes: 1000
+      data_preprocess:
+        mean:
+          - 123.675
+          - 116.28
+          - 103.53
+        std:
+          - 58.395
+          - 57.12
+          - 57.375
+        to_rgb: True
+  optimizer:
+    _target_: torch.optim.SGD
+    _partial_: true
+    lr: 0.007
+    momentum: 0.9
+    weight_decay: 0.0001
```

Comment:
For finetuning Transformers, we normally use 1) AdamW optimizer and 2) small lr ~ 1e-5 or 1e-4. It comes from my experiences.

Please see the following articles for more details:
1. https://www.montevideolabs.com/fine-tuning-a-vision-transformer/
2. https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/VisionTransformer/Fine_tuning_the_Vision_Transformer_on_CIFAR_10_with_the_%F0%9F%A4%97_Trainer.ipynb#scrollTo=8wTw4RXe73vA
3. https://arxiv.org/pdf/2106.10270.pdf

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2708/commits/7c8db3a54e84a4e183a48378fdd428e047288441

Reply:
`AdamW` and `Adam` are different each other. Is https://github.com/openvinotoolkit/training_extensions/commit/7c8db3a54e84a4e183a48378fdd428e047288441 using `Adam`?

Reply:
Ah, sorry. I mis-thinked that the default optimizer is the `AdamW`, thkanks for noticing

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2708/commits/686c15d2c9d2d025f01b02286cde6963a9aca7d9